app/src/main/java/com/glia/exampleapp/ExampleCustomCardAdapter.java

```diff
@@ -92,6 +99,16 @@ public NativeViewViewHolder(@NonNull View itemView, @NonNull UiTheme uiTheme) {
             this.okButton = itemView.findViewById(R.id.ok_button);
         }
 
+        public static boolean isNativeViewType(ChatMessage message) {
+            JSONObject metadata = message == null ? null : message.getMetadata();
+            if (metadata == null || metadata.length() == 0) {
+                return false;
+            } else {
+                return metadata.has(NativeViewViewHolder.SHOW_BUTTON_KEY)
```

Comment:
I am not sure that it is for these cases `NativeView` should be shown.
I'd better remove this custom view from our testing app.

Reply:
I think it is alright. But I can remove it as well if needed. @salemove/tm-mobile-android

Reply:
I mean that the `NativeView` is not necessary enough to leave it for such rather strange conditions.